src/index.css

```diff
@@ -1,3 +1,277 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;500;700&display=swap');
+
 body {
   box-sizing: border-box;
+  padding: 4%;
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
+  background-color: #f3f4f8;
+}
+
+#root {
+  background-color: #feffff;
+}
+
+header {
+  display: flex;
+  padding: 24px 4%;
+  justify-content: space-between;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+nav {
+  display: flex;
+  gap: 40px;
+}
+
+nav a {
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+}
+
+.header-title {
+  margin: 0;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+}
+
+.navbar-ul {
+  padding: 0;
+  list-style: none;
+  margin: 0;
+  display: flex;
+  gap: 40px;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-link {
+  font-size: 0.813rem;
+  color: #12121250;
+  letter-spacing: 1.9px;
+  text-transform: uppercase;
+}
+
+.active {
+  color: #121212;
+}
+
+.app-container {
+  background-color: #fafafa;
+  padding: 4%;
+}
+
+.user-icon {
+  color: #0290ff;
+}
+
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  padding: 2rem;
+  margin-bottom: 2.5rem;
+}
+
+.book-category {
+  margin: 0;
+  font-size: 0.875rem;
+  color: #888;
+}
+
+.book-name {
+  margin: 0.188rem 0;
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+  font-family: "Roboto Slab", serif;
+  font-weight: 700;
+  color: #121212;
+}
+
+.book-author {
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  margin: 0;
+  font-family: "Roboto Slab", serif;
+}
+
+.book-buttons-container {
+  display: flex;
+  align-items: center;
+  margin-top: 1.313rem;
+}
+
+.book-button {
+  background-color: transparent;
+  border: none;
+  padding: 0;
+  color: #4386bf;
```

Comment:
- [ ] Kindly do the same for this color property which is reapplied in line 114 by using a reusable class for CSS best practices

Reply:
Thanks for the suggestion. Fixed